pyproject.toml

```diff
@@ -5,7 +5,7 @@ build-backend = "setuptools.build_meta"
 [project]
 # please adapt to package name
 name = "ghga-transpiler"
-version = "1.3.0"
+version = "1.4.1"
```

Comment:
```suggestion
version = "1.4.0"
```
We can delete the 1.4.0 release and create again, the CD pipelines anyway didn't run

Reply:
We need to recreate the tag as well then